pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pkgs }:
+
+buildPythonPackage rec {
+  pname = "ovirt-engine-sdk-python";
+  version = "4.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hz7rcz7l391n8ixcwvqw3iq5bx26fk9l5c6xxymv036zm3r9pnh";
+  };
+
+  doCheck = false;
+
+  buildInputs = with pkgs.python37Packages; [ 
```

Comment:
Python packages are not often in `buildInputs`. This would imply the Python packages in this list are only needed during build-time and not run-time. Is that correct?

Reply:
you're right, not correct - I misunderstood the buildInputs behavior, will fix, thanks